### ISSUE TITLE
Creating a Binding

### ISSUE THREAD

**Issue by SimpleSamples** [dotnet/docs-desktop#180]
The "Creating a Binding" section takes a major leap over some critical topics. I know from my struggles that DataContext is important to understand and it is not adequately described here. Also, you guys like to use those "Resources" things and that is not adequately described here.

And for such a beginner level article such as this you really should include the MyData class.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 87567591-0d1a-a791-a609-fa7a3986d43c
* Version Independent ID: 29542eb2-683e-0217-71aa-14671a6f193a
* Content: [Data Binding Overview](https://docs.microsoft.com/en-us/dotnet/framework/wpf/data/data-binding-overview)
* Content Source: [docs/framework/wpf/data/data-binding-overview.md](https://github.com/dotnet/docs/blob/master/docs/framework/wpf/data/data-binding-overview.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mrmsbarnes** [comment]
Oh, to be groomed now that the camellias are blooming.

**Comment by adegeo** [comment]
I'm going to be revising this content for a publish soon, so I'll be trying to figure out what can be improved and help cross linking to other concepts.

**Comment by mrmsbarnes** [comment]
Thank you, Andy.  I know it’s old stuff.  But even just for reviewing (I do a lot), I think a clearer or more complete description would be very helpful.

Thanks and good luck.

**Comment by adegeo** [comment]
For this I'm going to update it a bit to hopefully make it a bit clearer. I moved some sections around and edited the introductory sections to be clearer. I created a new data context section too. Hopefully this helps. I'm going to plan on adding a new article that walks through creating databindings, more like an overview through code examples instead of what we have here which is a typical conceptual overview.
